v1/docker.yaml

```diff
@@ -0,0 +1,58 @@
+description: A playground environment with docker and related tools
+version: latest
+
+instances:
+  docker:
+    limits:
+      min-cpu: 2
+      min-mem: 4G
+      min-disk: 40G
+    cloud-init:
+      vendor-data: |
+        groups:
+        - docker
+        snap:
+          commands:
+          - snap install docker
+          - snap install yq
+
+        runcmd:
+        - |
+          # add `ubuntu` to the `docker` group
+          adduser ubuntu docker
+          
+        - |
+          # disable swap
+          sysctl vm.swappiness=0
+          echo "vm.swappiness = 0" | tee -a /etc/sysctl.conf
+
+        - |
+          # disable unnecessary services
+          systemctl disable man-db.timer man-db.service --now
+          systemctl disable apport.service apport-autoreport.service  --now
+          systemctl disable apt-daily.service apt-daily.timer --now
+          systemctl disable apt-daily-upgrade.service apt-daily-upgrade.timer --now
+          systemctl disable unattended-upgrades.service --now
+          systemctl disable motd-news.service motd-news.timer --now
+          systemctl disable bluetooth.target --now
+          systemctl disable ua-messaging.service ua-messaging.timer --now
+
+        - |
+          # various tools via apt
+          # gh cli
+          curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+          echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+
+          apt-get -y update
+          apt-get -y install jq skopeo gh
+
+        - |
+          # dive, to easily browse image layers
+          wget https://github.com/wagoodman/dive/releases/download/v0.9.2/dive_0.9.2_linux_amd64.deb
+          dpkg -i ./dive_0.9.2_linux_amd64.deb
```

Comment:
Is this workflow intended for more than just amd64?

Reply:
Yeah we'll need to use `$( dpkg-architecture -q DEB_HOST_ARCH )` in these cases. And maybe hit some API for the newest version available.

Reply:
The deb is only available for amd64, we would need to build it from source instead.

Reply:
Any chance it could be as easy as augmenting this list of `goarch`?
https://github.com/wagoodman/dive/blob/c7d121b3d72aeaded26d5731819afaf49b686df6/.goreleaser.yml#L13
Would need to somehow match it with `goos` though.

Reply:
I've dropped it for now, we've added portainer that allows for at least some of the same functions.